metapool/count.py

```diff
@@ -160,6 +169,25 @@ def bcl2fastq_counts(run_dir, sample_sheet):
     return out
 
 
+def _bclconvert_counts(path, sample_sheet):
+    # read the csv in from file
+    df = pd.read_csv(path)
+    # subselect only the columns we're concerned with
+    df = df[["SampleID", "Lane", "# Reads"]]
+    # filter out rows that reference and 'Undetermined' fastq.gz file
+    df = df.loc[df['SampleID'] != 'Undetermined']
+    # create our own copy to return to the user
+    df = df.copy()
```

Comment:
You can probably combine this with the .loc call before.

Reply:
Thanks! Will do!